src/page-styles 2/what-we-do-styles.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components"
```

Comment:
I think this is a duplicate folder lol. "page-styles" and "page-styles 2"

Reply:
ugh why does my stupid thing keep duplicating files 😭